src/main/cpp/profiler/spark_rapids_profile_converter.cpp

```diff
@@ -114,6 +116,8 @@ Converts the spark-rapids profile in profile.bin into other forms.
   -i, --json-indent=INDENT  indentation to use for JSON. 0 is no indent, less than 0 also removes newlines
   -o, --output=PATH         use PATH as the output filename
   -t. --nvtxt               convert to NVTXT, default output is stdout
+  -w. --nvtxw               generate nsys-rep using NVTXW API
+  --nvtxw-backend=PATH      use PATH for the NVTXW backend library
```

Comment:
What about the environment variable `NVTXW_BACKEND`? This application parameter is not mentioned anywhere in the docs thus I'm not sure if setting `NVTXW_BACKEND` env is enough.

Reply:
Both --nvtxw-backend or NVTXW_BACKEND env works to provide the NVTXW backend library path. Updated the doc.